src/main/java/com/contrastsecurity/maven/plugin/sdkx/GsonFactory.java

```diff
@@ -7,20 +7,37 @@
 import com.google.gson.stream.JsonWriter;
 import java.io.IOException;
 import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
 
 /** Factory for configuring an instance of GSON that is compatible with the Scan API */
 final class GsonFactory {
 
   static Gson create() {
     return new GsonBuilder()
-        .registerTypeAdapter(LocalDateTime.class, new LocalDateTimeTypeAdapter())
+        .registerTypeAdapter(LocalDateTime.class, new LocalDateTimeTypeAdapter().nullSafe())
+        .registerTypeAdapter(ZonedDateTime.class, new ZonedDateTimeTypeAdapter().nullSafe())
         .create();
   }
 
   /** static members only */
   private GsonFactory() {}
 
+  private static final class ZonedDateTimeTypeAdapter extends TypeAdapter<ZonedDateTime> {
+
+    @Override
+    public void write(final JsonWriter writer, final ZonedDateTime value) throws IOException {
+      final String formatted = value.format(DateTimeFormatter.ISO_ZONED_DATE_TIME);
+      writer.value(formatted);
+    }
+
+    @Override
+    public ZonedDateTime read(final JsonReader reader) throws IOException {
+      final String iso8601 = reader.nextString();
+      return ZonedDateTime.parse(iso8601, DateTimeFormatter.ISO_ZONED_DATE_TIME);
+    }
+  }
+
   private static final class LocalDateTimeTypeAdapter extends TypeAdapter<LocalDateTime> {
```

Comment:
I think we’ve determined that this isn’t necessary anymore right? Up to you if you want to remove it now or wait until later. I know you just asked about this in slack, so just putting a comment here as a reminder.

Reply:
Yes, the Scan API team informed me that they fixed the API. I'll get rid of the `LocalDateTime` stuff. I think I'm wrong to use `ZonedDateTime` - I should be using `OffsetDateTime` - so I'll make that change too.